Factorial

```diff
@@ -1,14 +1,27 @@
 #include <iostream>
 
-int main()
+int factorial(int n)
```

Comment:
Please, also solve merge conflicts with master branch
https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-on-github